docs/dev/new_check_howto.md

```diff
@@ -74,7 +77,7 @@ This will display the path where the files would have been written, as well as t
 The interactive mode is a wizard for creating new Integrations. By answering a handful of questions, the scaffolding will be set up and lightly pre-configured for you.
 
 ```
-ddev -e create my_check
+ddev create awesome
```

Comment:
This addresses the issue of accidentally creating `MyCheckCheck` which is nice. 👍